letsencrypt/cli.py

```diff
@@ -669,7 +682,12 @@ def determine_verb(self):
 
         for i, token in enumerate(self.args):
             if token in self.VERBS:
-                self.verb = token
+                verb = token
+                if verb == "certonly":
+                    verb = "auth"
```

Comment:
I don't like having aliases. Let's just use `certonly` and get rid of `auth` altogether. It makes the code complicated, and is harder to grasp from UX point of view.